src/client/components/AdminPanel/BruincastAdminListings.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from 'react';
+
+import { View } from '@instructure/ui-view';
+import { Text } from '@instructure/ui-text';
+import { Button } from '@instructure/ui-buttons';
+import { Grid } from '@instructure/ui-grid';
+import { TextInput } from '@instructure/ui-text-input';
+import { ScreenReaderContent } from '@instructure/ui-a11y-content';
+import axios from 'axios';
+
+import { BruincastAdminListingsToggle } from './BruincastAdminListingsToggle';
+import { ltikPromise } from '../../services/ltik';
+
+export const BruincastAdminListings = () => {
+  const [searchTerm, setSearchTerm] = useState('');
+  const [mediaListings, setMediaListings] = useState([]);
+
+  const handleTermInput = event => {
+    setSearchTerm(event.target.value);
+  };
+
+  const retrieveCastListings = () => {
+    ltikPromise.then(ltik => {
+      axios
+        .get(`/api/medias/bruincast/castlistings?ltik=${ltik}`, {
+          params: { term: searchTerm },
+        })
+        .then(res => setMediaListings(res.data));
+    });
+  };
+  useEffect(retrieveCastListings, []);
+
+  const searchWithTerms = async () => {
```

Comment:
Why do we need this function? Can't we just use retrieveCastListings?

Reply:
Whoops — forgot to remove that when refactoring the code